pkg/fetcher/definition.go

```diff
@@ -22,33 +23,71 @@ type Definition struct {
 	reader client.Reader
 }
 
+type uniqueTestDefinitions map[types.UID]v1alpha1.TestDefinition
+
 func (s *Definition) FindMatching(suite v1alpha1.ClusterTestSuite) ([]v1alpha1.TestDefinition, error) {
 	ctx := context.TODO()
-	if len(suite.Spec.Selectors.MatchNames) > 0 {
-		return s.findByNames(ctx, suite)
+	acc := make(uniqueTestDefinitions)
+
+	err := s.findByNames(ctx, suite, acc)
+	if err != nil {
+		return nil, err
 	}
-	// TODO(aszecowka) later so far we return all test definitions for all namespaces (https://github.com/kyma-incubator/octopus/issues/7)
+
+	err = s.findByLabelExpressions(ctx, suite, acc)
```

Comment:
general question - is it ok to add results of label expressions to the list of tests which matches by name? is it described somewhere in the documentation?

Reply:
I think it is ok, but we have to add that to documentation.

Reply:
It is mentioned in suite docs: 
```
| **spec.selectors** | **NO** | Defines which tests should be executed. You can define tests by specifying their names or labels. Selectors are additive. If not defined, all tests from all Namespaces are executed.
```